src/utils/docs.jl

```diff
@@ -162,9 +181,12 @@ $(VERSION). Running the doctests will produce errors that you do not expect."
   Pkg.activate(docsproject) do
     Base.invokelatest(Main.BuildDoc.doit, Oscar; strict=strict, local_build=true, doctest=doctest)
   end
-  if open_browser
+  if open_browser && !start_server
     open_doc()
   end
+  if start_server
+    start_doc_preview_server(open_browser = open_browser)
+  end
```

Comment:
IMHO this would be more natural, but it's a matter of taste, we can also leave it *shrug*
```suggestion
  if start_server
    start_doc_preview_server(open_browser = open_browser)
  elseif open_browser
    open_doc()
  end
```